### PR TITLE
Fcrepo 1290

### PR DESCRIPTION
**Pull request by yinlinchen**
Add return nonRdfSource properties in response and test case template

### CODE REVIEW

File: fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/PropertiesRdfContext.java [hunk 1]
```diff
@@ -56,19 +56,25 @@
      */
 
     public PropertiesRdfContext(final FedoraResource resource,
-                                final IdentifierConverter<Resource, FedoraResource> idTranslator)
```
Comment (awoods) [inline fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/PropertiesRdfContext.java]: As a side note, please do not include unrelated formatting updates in pull-requests.

Comment (yinlinchen) [inline fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/PropertiesRdfContext.java]: line 59? It was formatted by Eclipse using the auto style check using the https://github.com/fcrepo4/fcrepo4/tree/master/src/site/eclipse and I run mvn checkstyle:check before submit PR.

Comment (awoods) [inline fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/PropertiesRdfContext.java]: In any case, please do not include non-related formatting updates.
